modules/enterprise/server/jar/src/main/java/org/rhq/enterprise/server/content/ContentManagerBean.java

```diff
@@ -223,6 +224,12 @@ public void mergeDiscoveredPackages(ContentDiscoveryReport report) {
             LOG.debug("Finished merging [" + report.getDeployedPackages().size() + "] packages in "
                 + (System.currentTimeMillis() - start) + "ms");
         }
+
+        try{
+            DataPurgeJob.purgeNow();
```

Comment:
I guess that is OK for quick testing, but we might prefer to wait for the DataPurgeJob to be executed. AFAIK it executes every hour or so by default.